.github/CODEOWNERS

```diff
@@ -147,3 +147,26 @@ Dockerfile @juspay/hyperswitch-infra
 docker-compose.yml @juspay/hyperswitch-infra
 .dockerignore @juspay/hyperswitch-infra
 monitoring/ @juspay/hyperswitch-infra
+
+crates/router/src/core/revenue_recovery  @juspay/hyperswitch-revenue-recovery
+crates/router/src/core/payments/operations/proxy_payments_intent.rs @juspay/hyperswitch-revenue-recovery
+crates/router/src/core/payments/operations/payment_attempt_record.rs    @juspay/hyperswitch-revenue-recovery
```

Comment:
Nit: You can reduce the number of spaces in lines like this where there are multiple spaces.